README.md

```diff
@@ -18,15 +18,50 @@ This repository is for the organization and governance of the Tessel project as
 
 For an introduction to the project's organization and key repos, check out the [Technical Overview](https://tessel.gitbooks.io/t2-docs/content/Debugging/Technical_Overview.html).
 
-## How can I get involved?
+# Current goals
 
-### Quick start
+On September 2, 2016, the Steering Committee set year-long goals. They are organized into two overarching goals, each with several supporting goals:
+
+1. **Grow the community for development, inclusion, accessibility**
+  * [ ] Build effective working groups that can complete their tasks with measurable results. Celebrate the work of each of the working groups as goals are accomplished
+  * [ ] Upgrade our documentation. Build more fritzing examples, API prototypes, and call out features that are currently missing from the docs
+  * [ ] Update the tessel.io website to more accurately represent the project and its plans (https://github.com/tessel/tessel.io/issues/102)
+2. **Demonstrate projects and create new tools that prove Tessel is a production-ready (robust, reliable, and extensible) platform for modern hardware development**
+  * [ ] Create a guide to production-scale deployment of a Tessel project
+  * [ ] Research user needs: What do people need to use T2 in production? Why aren't they using it currently?
+  * [ ] Build Reach (https://github.com/tessel/project/issues/142)
+  * [ ] Get to JavaScript parity with Rust API and documentation (https://github.com/tessel/tessel-rust). Figure out JS-Rust inter-exection
+  * [ ] Investigate and possibly execute a Tessel hardware upgrade (Tessel 2.1) for more RAM and Flash. This should legitimize the platform and build out the capabilities so there is less user experience friction
+
+## Working groups
+
+To enable the above goals, the Tessel Project is establishing working groups.
+
+### What's a working group?
+* A working group (WG) is a group of people working toward a defined, accomplishable goal.
+* WGs create some output measure of progress on a regular (every 1-2 weeks) schedule, whether that's meeting notes or something else
+
+### What working groups do we need to accomplish this year's goals?
+* **Website WG**:  Create a better website for what we are & what we plan to be based on this year’s goals
+* **Rust WG**: Get Rust to 1st class support
+* **Reach WG**: ship Reach
+* **Learning WG**: investigate user needs for production & write a guide on how to take a Tessel project to product scale. Also build and document GPIO/GUTS (Great Uses for Tessel in Stuff e.g. hacking a production system) projects
+* **Tessel space WG**: make larger projects deployable on T2, possibly as a 2.1 Tessel hardware
+
+### How do we implement these WGs?
+We don't want to kick off all 5 at once, so we've picked 2 as pilots: website WG and Learning WG. These two are selected because they are reasonably well defined, accessible, and don't require specialized skills.
+
+We will bring up the other WGs as we become comfortable with the process.
```

Comment:
(Other WGs will be formed as we learn from working group progress and as interested contributors come on board. Reach out to us if you're interested!)